dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -310,17 +311,19 @@ public void saveStep(WorkflowStep step) throws DotDataException, AlreadyExistExc
 	@WrapInTransaction
 	public void deleteStep(final WorkflowStep step) throws DotDataException {
 
+		Savepoint savepoint = null;
 		try {
 
 			// Checking for Next Step references
 			for(WorkflowStep otherStep : findSteps(findScheme(step.getSchemeId()))){
-				if(otherStep.equals(step))
-					continue;
+				//if(otherStep.equals(step))

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
DONE